private/src/styles/blocks/postlist/_categories.scss

```
@@ -10,7 +10,7 @@
 .postlist-categories {
   display: flex;
   flex: 1 1 100%;
-  padding: 20px;
+  padding: 20px !important;
```

we should probably use the theme.json stuff, now that it's available.
```suggestion
  padding: var(--wp--preset--spacing--single) !important;
```